README.md

```diff
@@ -1,2 +1,90 @@
-# reflex-website
-A public repo of the reflex.dev website.
+<div align="center">
+<img src="https://raw.githubusercontent.com/reflex-dev/reflex/main/docs/images/reflex_dark.svg#gh-light-mode-only" alt="Reflex Logo" width="300px">
+<img src="https://raw.githubusercontent.com/reflex-dev/reflex/main/docs/images/reflex_light.svg#gh-dark-mode-only" alt="Reflex Logo" width="300px">
+
+<hr>
+
+### **✨ Performant, customizable web apps in pure Python. Deploy in seconds. ✨**
+[![PyPI version](https://badge.fury.io/py/reflex.svg)](https://badge.fury.io/py/reflex)
+![tests](https://github.com/pynecone-io/pynecone/actions/workflows/integration.yml/badge.svg)
+![versions](https://img.shields.io/pypi/pyversions/reflex.svg)
+[![Documentation](https://img.shields.io/badge/Documentation%20-Introduction%20-%20%23007ec6)](https://reflex.dev/docs/getting-started/introduction)
+[![Discord](https://img.shields.io/discord/1029853095527727165?color=%237289da&label=Discord)](https://discord.gg/T5WSbC2YtQ)
+</div>
+
+# Overview
+
+A public repository which contains the source code of the reflex.dev website.
+
+- It's built with [Reflex](https://reflex.dev/), a Python web framework.
+- You can find everything related to Reflex at [reflex.dev](https://reflex.dev/).
+
+## About Reflex
+
+Reflex.dev is a groundbreaking web development framework that empowers Python developers to seamlessly create web apps without the nuances of traditional frontend tools. Its open-source core framework ensures flexibility for projects of any scale. Explore the future of web development at [reflex.dev](https://reflex.dev/).
```

Comment:
For the first sentence I think we don't need to say "groundbreaking" let's just make it something like "Reflex is a full-stack web framework that allows developers to build their app in pure Python, and deploy with a single command".